stage2/cc_amd64.s

```diff
@@ -354,8 +354,7 @@
 	JUMP @get_token_done        ; Finish off the token
 
 	;; Looks like it was //
-	FGETC                       ; Get next char
-	JUMP @get_token_reset       ; Try again
+	JUMP @purge_macro           ; Try again
```

Comment:
Update this comment too. In other files too

Reply:
Fixed.